azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/StorageService.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.domain;
+
+import com.google.auto.value.AutoValue;
+import java.util.Date;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableMap;
+import org.jclouds.javax.annotation.Nullable;
+import org.jclouds.json.SerializedNames;
+
+@AutoValue
+public abstract class StorageService {
+
+   public enum AccountType {
+
+      Standard_LRS,
+      Standard_ZRS,
+      Standard_GRS,
+      Standard_RAGRS,
+      Premium_LRS,
+      UNRECOGNIZED;
+
+      public static AccountType fromString(final String text) {
```

Comment:
Better rename to `fromValue` so we can benefit from it during deserialization. jclouds will call the `fromValue` when deserializing values if the default `enum.valueOf` fails (see [EnumTypeAdapterThatReturnsFromValue](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/json/internal/EnumTypeAdapterThatReturnsFromValue.java), the default enum type adapter used in jclouds).

I've just noticed this, but it would be great if you could rename any other similar methods in the enums introduced in the other ARM PRs, if any.